tests/integration/test_nvt_tf_inference.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import warnings
+
+import cudf
+import cupy as cp
+import numpy as np
+import pytest
+import tritonclient.grpc as grpcclient
+from tritonclient.utils import np_to_triton_dtype
+
+import nvtabular as nvt
+
+
+# Update TEST_N_ROWS param in test_nvt_tf_trainin.py to test larger sizes
+@pytest.mark.parametrize("n_rows", [64, 45, 32, 14, 7, 1])
+@pytest.mark.parametrize("err_tol", [0.00001])
+def test_nvt_tf_movielens_inference(n_rows, err_tol):
```

Comment:
This seems to require that tritonserver is already started with the model, which complicates how this will be run.

We do have some code in the unittests that starts the tritonserver process and waits for it to start up before running the test: https://github.com/NVIDIA/NVTabular/blob/9e14acc4eeb103bdd9590fb2a28eb5a182151be2/tests/unit/test_triton_inference.py#L23-L45 . 

At a minimum I think we need a shell script (potentially in ci/ folder) that shows how to run these tests. The inference integration tests are currently configured only in jenkins which not everyone has access to. @albert17 fyi.

Reply:
@albert17 Let's work together on whatever is needed.

Reply:
@benfred @oyilmaz-nvidia What I do right now for the existing integration tests is to launch the triton server before running the integration tests.

Fort unit tests we could use the same approach modifying the CI script.

Yes, for integration I need to add the scripts under CI folder. I keep forgetting. Let me try to put a PR today. Sorry

Reply:
@albert17 I also need some help on how to get the running time measurements to the CI.

And, once this gets merged, we'll need to run this on CI.

Reply:
@benfred @oyilmaz-nvidia sorry for the delay. https://github.com/NVIDIA/NVTabular/pull/863 I will ping you for the measurements

Reply:
@oyilmaz-nvidia Actually I will reply here for measurements. 

We need to use a benachmark in the similar way that criteo test has one, for example: https://github.com/NVIDIA/NVTabular/blob/main/tests/integration/test_notebooks.py#L69

For each benchmark there is a Class, for example for Criteo FastAI: https://github.com/NVIDIA/NVTabular/blob/main/tests/integration/criteo_parsers.py#L4

So we need to create one.

Reply:
@albert17 Updated the tests and now we have two main tests (test_nvt_tf_rossmann_inference_triton and test_nvt_tf_rossmann_inference_triton_mt) that take time measurements. I called the send_benchmark function but getting error. Actually, I don't know how to call the tests that send benchmark results for CI. Can you please help me with this?